src/main/Bot.go

```diff
@@ -106,6 +110,8 @@ func main() {
 	img_client.Log = new(klogger.CLILogger)
 	img_client.ImgurClientID = ImgurID
 
+	OsuClient = *osuapi.NewClient(OsuID)
```

Comment:
We should really keep this consistent. I used img_client for the Imgur API, so we should either change to to ImgClient or this to osu_client. (also Osu, triggered)